### PR TITLE
feat: add manual verification

### PR DESCRIPTION
**Pull request by builtbyvys**

### CODE REVIEW

File: src/commands/verify.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    if (targetMember.roles.cache.has(verifiedRole.id)) {
+      return interaction.reply({
+        content: `${targetUser.username} is already verified.`,
+        ephemeral: true,
+      });
+    }
+
+    await db.setData(targetUser.id, ip);
+    await targetMember.roles.add(verifiedRole);
```
Comment (builtbyvys) [inline src/commands/verify.js:66]: this currently isn't working at the moment (postgres gives me a headache)

Comment (Rein1n) [reply src/commands/verify.js]: Your local setup might be messed up, cause it's working for me.

**Review by Rein1n — COMMENTED** [review]
Utilize the "integration" tab in server settings to set native permissions.

File: src/commands/verify.js [hunk 2]
```diff
@@ -23,14 +23,6 @@ export default {
         .setRequired(true)
     ),
   async execute(interaction) {
-    const member = await interaction.guild.members.fetch(interaction.user.id);
-    if (!member.roles.cache.has(process.env.STAFF_ROLE)) {
-      return interaction.reply({
-        content: "You do not have permission to use this command.",
-        ephemeral: true,
-      });
-    }
```
Comment (Rein1n) [inline src/commands/verify.js]: Utilize integrations tab on server settings for native permissions

**Pull request merged**